### ISSUE TITLE
Merge mobility analysis with master

### ISSUE THREAD

**Issue by jdehning** [Priesemann-Group/covid19_inference#17]
@joaopn What is the status there? Otherwise I think that @semohr has also some code to retrieve the mobility data. I just want to avoid diverging development

**Comment by joaopn** [comment]
Absolutely, I'll definitely use @semohr 's code =P Once code is mature I'll spread it back to the main modules, but I'd keep the development in a separate branch for now.

**Comment by jdehning** [comment]
Good :+1:
